.github/workflows/ci.yaml

```diff
@@ -300,23 +254,20 @@ jobs:
         with:
           fetch-depth: 1
 
-      - name: Setup Go
-        uses: ./.github/actions/setup-go
-
-      - name: Setup Terraform
-        uses: ./.github/actions/setup-tf
+      - name: Setup Nix
```

Comment:
Not using container heer as we need to run a Postgres container to run tests. I am not sure if it is possible to achieve this using GitHub Actions. i.e container in container
So I am directly using `nix-shell --command` here.

same applies to `gen` job